docs/sources/operations/query-fairness/_index.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Query fairness within tenants
+menuTitle: Query fairness
+description: The scheduler can guarantee query fairness across multiple actors within a single tenant.
+weight: 101
+---
+
+# Query fairness within tenants
+
+Loki uses [shuffle sharding]({{<relref "../shuffle-sharding/_index.md">}})
+to minimize impact across tenants in case of querier failures or misbehaving
+neighbouring tenants. However, there are use-cases where a Loki installation
+only has a handful of very large tenants with a lot of actors (humans,
+applications, ...).
+
+When there are potentially a lot of different users using the same tenant to
+query logs, such as users accessing Loki from Grafana or via LogCLI or other
+applications using the HTTP API, it can lead to contention between queries of
+different users, because they all share the same tenant.
+
+In that case, as an operator, you would also want to ensure some sort of query
+fairness across these actors within the tenants. To achieve that, Loki
+introduced hierarchical scheduler queues in version 2.9 and they are enabled by
+default.
```

Comment:
Should we link the LID here so folks can read up on the design decisions?

Reply:
Yes